core/src/banking_stage/immutable_deserialized_packet.rs

```diff
@@ -64,7 +65,8 @@ impl ImmutableDeserializedPacket {
         } = process_compute_budget_instructions(
             sanitized_transaction
                 .get_message()
-                .program_instructions_iter(),
+                .program_instructions_iter()
+                .map(|(pubkey, ix)| (pubkey, SVMInstruction::from(ix))),
```

Comment:
it's `SanitizedVersionedMessage` that doesn't impl `SVMMessage` trait, doing `map` directly.